pkg/cmd/pipelinerun/list_test.go

```diff
@@ -339,6 +382,33 @@ func TestListPipelineRuns_v1beta1(t *testing.T) {
 		t.Errorf("unable to create dynamic client: %v", err)
 	}
 
+	tdc2 := testDynamic.Options{
+		PrependReactors: []testDynamic.PrependOpt{
+			{
+				Verb:     "list",
+				Resource: "pipelineruns",
+				Action: func(action k8stest.Action) (bool, runtime.Object, error) {
+					if action.(k8stest.ListAction).GetListRestrictions().Fields.String() == "metadata.name!=pr0-1" {
```

Comment:
Extract (k8stest.ListAction).GetListRestrictions().Fields.String() a variable and then check.